modules/cloudbuild/README.md

```diff
@@ -59,8 +59,10 @@ Functional examples and sample Cloud Build definitions are included in the [exam
 | folder\_id | The ID of a folder to host this project | string | `""` | no |
 | group\_org\_admins | Google Group for GCP Organization Administrators | string | n/a | yes |
 | org\_id | GCP Organization ID | string | n/a | yes |
+| project\_labels | Labels to apply to the project. | map(string) | `<map>` | no |
```

Comment:
As discussed, can we remove this from the example as it is not required for the cloudbuild module:
```hcl
group_billing_admins    = "gcp-billing-admins@example.com"
```

Reply:
Revolved by 3bcba654e6b04dafffd1ba22b686158e4af05b8e.